manual/server/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* STOCKS-111: Add Debian packaging
+    * Remove nginx-reload, now handled via systemd
+    * DH parameter generation is now optional
```

Comment:
Hmm this reads strange now. I would change my former suggestion and split into two points:
* Add debian packaging
* Replace nginx-reload by direct systemd command
* (Don't talk about dhparams. Either the installation already exists and has dhparam -> no changes and admin doesn't care. Or it is a fresh installation so it never knew that dhparams were ever present)

Reply:
Both should reference `STOCKS-111` then I assume?

Reply:
Exactly